src/utils/validate.js

```diff
@@ -0,0 +1,74 @@
+import { MESSAGE } from '../constants/message.js'
+
+class validate {
+	inputMoney(inputMoney) {
+		this.#isNotNumber(inputMoney)
+		this.#isZeroOrminus(inputMoney)
```

Comment:
```suggestion
		this.#isZeroOrMinus(inputMoney)
```
`sZeroOrminus` 뭐지 싶었는데 `isZeroOrMinus` 였군요! 중간에 카멜 케이스가 빠져서 `Orminus` 이렇게 한 단어인줄 알았네요🤣

Reply:
앜ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ그러게욯ㅎㅎㅎ 저도 보면서 저게 무슨 클래스였지...? 저또한 못읽었네요! 리팩토링때 수정해야겠군요